bug-bounty-hof/client.yml

```diff
@@ -1,4 +1,123 @@
 names:
+- name: David Klein
+  date: 2022-09-21
+- name: Meet Narkhede
+  date: 2022-09-20
+- name: Mario E. Mejia
+  date: 2022-09-20
+  twitter: "@Moblig_"
+- name: Anonymous
+  date: 2022-09-12
+  twitter: "@r3dpars3c"
+- name: Codermak
+  date: 2022-08-30
+  twitter: "@arshadkazmi42"
+- name: Muhammad R. Maulana
+  date: 2022-08-22
+  twitter: "@agamimaulana"
+- name: musicDespiteEverything
+  date: 2022-08-22
+- name: Irwan
+  date: 2022-08-01
+- name: Brandon Roldan
+  date: 2022-07-11
+  twitter: "@tomorrowisnew"
+  url: https://twitter.com/@tomorrowisnew
+- name: Axel Chong
+  date: 2022-09-20
+- name: Andrei Enache
+  date: 2022-09-20
+- name: Armin Ebert
+  date: 2022-09-20
+  url: https://rawsec.net/
+- name: Irvan Kurniawan (sourc7)
+  date: 2022-08-23
+  url: https://www.linkedin.com/in/sourc7/
+- name: Abraruddin Khan and Omair
+  date: 2022-08-23
+  url: https://krashconsulting.com
+- name: Muhammad R. Maulana
+  date: 2022-08-22
+  twitter: "@agamimaulana"
+  url: https://twitter.com/@agamimaulana
+- name: Gary Kwong
+  date: 2022-07-28
+  url: https://www.ygitsoftware.com/
+- name: Muneaki Nishimura (a.k.a. nishimunea)
+  date: 2022-07-19
+- name: Young Min Kim (CompSec Lab at Seoul National University)
+  date: 2022-07-19
+  twitter: "@ylemkimon"
+  url: https://twitter.com/@ylemkimon
+- name: akucybersec
+  date: 2022-07-12
+- name: Gustavo Grieco
+  date: 2022-07-05
+- name: Kirin of Tencent Security Xuanwu Lab
+  date: 2022-07-05
+  twitter: "@Pwnrin"
+  url: https://twitter.com/@Pwnrin
+- name: Muneaki Nishimura (a.k.a. nishimunea)
+  date: 2022-06-28
+- name: Irwan
+  date: 2022-06-28
+- name: Young Min Kim (CompSec Lab at Seoul National University)
+  date: 2022-06-14
+  twitter: "@ylemkimon"
```

Comment:
Is this a duplicate of the ylemkimon on lines 48-51? Or is it necessary because those dates are in two different quarters?

Reply:
Different quarters - June is Q2, July is Q3

Reply:
yes I was planning to prepare a PR once we merge this one to also avoid conflicts